src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -56,52 +54,57 @@ public PID(double P, double I, double D, int slot) {
         }
     }
 
-    // Position Constants

```

Comment:
You'll need to put these back if you're following my other comments or at least update the other code to use your enum based solution which I am not apposed to though what you're referencing exactly will likely get obscured. This can be discussed.